crates/nu-command/src/help/help_operators.rs

```diff
@@ -27,282 +28,144 @@ impl Command for HelpOperators {
+        Operator::Comparison(Comparison::NotEqual) => "Checks if two values are not equal.",
+        Operator::Comparison(Comparison::LessThan) => "Checks if a value is less than another.",
+        Operator::Comparison(Comparison::GreaterThan) => {
+            "Checks if a value is greater than another."
+        }
+        Operator::Comparison(Comparison::LessThanOrEqual) => {
+            "Checks if a value is less than or equal to another."
+        }
+        Operator::Comparison(Comparison::GreaterThanOrEqual) => {
+            "Checks if a value is greater than or equal to another."
+        }
+        Operator::Comparison(Comparison::RegexMatch) => {
+            "Checks if a value matches a regular expression."
+        }
+        Operator::Comparison(Comparison::NotRegexMatch) => {
+            "Checks if a value does not match a regular expression."
+        }
+        Operator::Comparison(Comparison::In) => {
+            "Checks if a value is in a list, is part of a string, or is a key in a record."
+        }
+        Operator::Comparison(Comparison::NotIn) => {
+            "Checks if a value is not in a list, is not part of a string, or is not a key in a record."
+        }
+        Operator::Comparison(Comparison::StartsWith) => "Checks if a string starts with another.",
+        Operator::Comparison(Comparison::EndsWith) => "Checks if a string ends with another.",
+        Operator::Math(Math::Plus) => "Adds two values.",
+        Operator::Math(Math::Append) => "Appends two lists, a list and a value, two strings, or two binary values.",
+        Operator::Math(Math::Minus) => "Subtracts two values.",
+        Operator::Math(Math::Multiply) => "Multiplies two values.",
+        Operator::Math(Math::Divide) => "Divides two values.",
+        Operator::Math(Math::Modulo) => "Divides two values and returns the remainder.",
+        Operator::Math(Math::FloorDivision) => "Divides two values and floors the result.",
+        Operator::Math(Math::Pow) => "Raises one value to the power of another.",
+        Operator::Boolean(Boolean::And) => "Checks if both values are true.",
+        Operator::Boolean(Boolean::Or) => "Checks if either value is true.",
+        Operator::Boolean(Boolean::Xor) => "Checks if one value is true and the other is false.",
+        Operator::Bits(Bits::BitOr) => "Performs a bitwise OR on two values.",
+        Operator::Bits(Bits::BitXor) => "Performs a bitwise XOR on two values.",
+        Operator::Bits(Bits::BitAnd) => "Performs a bitwise AND on two values.",
+        Operator::Bits(Bits::ShiftLeft) => "Bitwise shifts a value left by another.",
+        Operator::Bits(Bits::ShiftRight) => "Bitwise shifts a value right by another.",
+        Operator::Assignment(Assignment::Assign) => "Assigns a value to a variable.",
+        Operator::Assignment(Assignment::PlusAssign) => "Adds a value to a variable.",
+        Operator::Assignment(Assignment::AppendAssign) => "Appends a value to a variable.",
```

Comment:
I'm not sure why this needs to change because it still seems to work with a list or a value.
```suggestion
        Operator::Assignment(Assignment::AppendAssign) => "Appends a list or value to a variable.",
```

Reply:
A fully correct description would be:

> Appends a list or a value to a list variable, a string to a string variable, or a binary value to a binary variable.

So, I just shortened it instead :shrug:

Maybe
> Appends a list, a value, a string, or a binary value to a variable.

would be better?

Reply:
ya, i think your last suggestion is good.